src/Sylius/Bundle/ApiBundle/Command/VerifyCustomerAccount.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+/**
+ * @experimental
+ * @psalm-immutable
+ */
+class VerifyCustomerAccount
```

Comment:
Choose one form, because you are using both customer and shop user. Here is customer, but in other command you have shop user: `SendShopUserVerificationEmail`